doc/source/extended_documentation/calibration/ensemble_calibration/ensemble_calibration.rst

```diff
@@ -82,6 +82,29 @@ The scale parameter indicates the width in the distribution. If the scale
 parameter is large, then the distribution will be broader. If the scale is
 smaller, then the distribution will be narrower.
 
+****************************************************
```

Comment:
I think there are a few places higher up where scale parameter should be replaced with scale parameter squared

Reply:
I've edited this in one place above in this file.